dags/config/gcs_template_source.yaml

```diff
@@ -37,8 +37,9 @@ templates:
       file_table_config:
         directory_prefix: ""
         allow_quoted_newlines: True
+    iterable_options: []
```

Comment:
can we make [] a default. This way a user doesn't need to provide this option at all if they don't care

Reply:
I would also validate that iterable_options can have a non-empty value only if table_names is provided (it doesn't work with any of the other options, like reg-ex matching)